src/serlo-editor/plugins/image/editor.tsx

```diff
@@ -92,17 +103,4 @@ export function ImageEditor(props: ImageProps) {
       } as TextEditorConfig,
     })
   }
-
-  function renderEditControls() {
```

Comment:
I'm a bit confused, I thought you preferred this `render*` approach, but in this plugin toolbar PR, you've been removing a lot of them 😄 What's your preference now? I'd like to align my coding style with yours where possible

Reply:
(see slack)